pkg/scaffold/internal/templates/v2/makefile.go

```diff
@@ -132,4 +134,19 @@ CONTROLLER_GEN=$(GOBIN)/controller-gen
 else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
+
+kustomize:
+ifeq (, $(shell which kustomize))
+	@{ \
+	set -e ;\
+	KUSTOMIZE_GEN_TMP_DIR=$$(mktemp -d) ;\
+	cd $$KUSTOMIZE_GEN_TMP_DIR ;\
+	go mod init tmp ;\
+	go get sigs.k8s.io/kustomize/kustomize/v3@{{ .KustomizeVersion }} ;\
+	rm -rf $$KUSTOMIZE_GEN_TMP_DIR ;\
+	}
+KUSTOMIZE=$(GOBIN)/kustomize
```

Comment:
What if the `GOBIN` is not set?
`go get sigs.k8s.io/kustomize/kustomize/v3@v3.5.4` works for me with go 1.14 with `GOPATH` and `GO111MODULE=on` set, but `GOBIN` unset.

Reply:
@mengqiy it is exactly the same impl for cotroller-gen. So, because of this I think would make sense as well. Also, see: https://github.com/kubernetes-sigs/kubebuilder/blob/7c6dc784a0c212a4f45e6a7f630ffced3c51ab57/pkg/scaffold/internal/templates/v2/makefile.go#L63-L68